server/auth/index.js

```diff
@@ -14,7 +14,7 @@ let sessionStrategy = {
       server.auth.strategy('session', 'cookie', config);
       server.auth.default({
         strategy: 'session',
-        scope: 'user'
+        scope: ['admin', 'user']
```

Comment:
Default scope should just be user, in case a developer forgets to add custom auth, to save from accidental privilege escalation.